docs/stellar-core_example.cfg

```diff
@@ -138,11 +138,11 @@ MAX_BATCH_WRITE_COUNT=1024
 # How many bytes can this server send at once to a peer
 MAX_BATCH_WRITE_BYTES=1048576
 
-# FLOOD_OP_RATE_PER_LEDGER (Integer) default -2
+# FLOOD_OP_RATE_PER_LEDGER (Floating point) default -1.0
 # How many operations get flooded per ledger
 # If negative, the number is derived as:
 #  -FLOOD_OP_RATE_PER_LEDGER*<maximum number of operations per ledger>
-FLOOD_OP_RATE_PER_LEDGER = -2
+FLOOD_OP_RATE_PER_LEDGER = -1.0
```

Comment:
Maybe we should just eliminate the possibility of "fixed number" flooding (eg. if this is positive) and always make it multiplicative? It's really strange to configure the fixed number as a floating point.

Reply:
agree, removed the fixed version that we don't need